tsconfig.settings.json

```diff
@@ -8,7 +8,11 @@
     "strict": true,
     "composite": true,
     "esModuleInterop": true,
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "lib": [
+      "esnext",
+      "es2017.object"
```

Comment:
Remove this, let's not use stuff that requires polyfills (except maybe promises).